play/square-wars/script.js

```diff
@@ -0,0 +1,715 @@
+//global objects for the game
+let player, enemy, enemys, bullet, bullets, boss, gameHandler, mathHandler;
+//global boolean for the game engine
+let gamePlaying;
+
+//difficulty level for game preset to 1
+let difficultyLevel = 1
+
+/**
+* used to set the difficulty level from the html main menu buttons
+* difficultyLevel is used to set up the MathClassHandler
+* @param {String} Level
+**/
+function setLevel(level) {
+  intLevel = parseInt(level)
+  difficultyLevel = intLevel;
+  document.getElementById("gameBtn").style.display = "inline";
+}
+
+/**
+* Fucntion to validate the numbers only in the input text box
+* @param (Event) evt
+**/
+function isNumber(evt) {
+  var iKeyCode = (evt.which) ? evt.which : evt.keyCode
```

Comment:
Use ```let```! Also ```evt.key``` or ```evt.code``` are better to use because they are more descriptive for key names. Check out: [https://keycode.info/](https://keycode.info/).